src/layman/layer/filesystem/input_chunk.py

```diff
@@ -131,6 +130,17 @@ def get_info_json(username, layername):
     return result
 
 
+def get_layer_info(workspace, layername):
```

Comment:
Add `'file'` item to `'layman.layer.filesystem.input_chunk'` source in layer type definition